src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -111,18 +105,18 @@ limitations under the License.
     <item name="type" value="0x0C"/>
   </enum>
 
-  <enum name="ContentLaunchStatus" type="ENUM8">
+  <enum name="StatusEnum" type="ENUM8">
```

Comment:
This is going to cause problems: I don't think ZAP and the helpers we use for codegen can deal with two enums with identical names, and we have that now for "StatusEnum", right?

Reply:
I've stopped using `<app-common/enums.h> `
I'm using `<app-common/cluster-objects.cpp> `. There I can use namespaced class enum type. Is that ok?

_The problem you've mentioned is exactly why I've prefixed every parameter with cluster name. I've had issues with duplicated data types._

Reply:
The C++ deals fine due to namespacing.  But ZAP itself does not.

Specifically, when you say `type="StatusEnum"` in your XML, which of the enums of that name do you mean?  ZAP has no idea, as far as I know, and it will more or less pick one randomly, I suspect.

@tecimovic Please let me know if that's not correct; I would be very happy to be wrong here.